mne/decoding/time_gen.py

```diff
@@ -109,6 +108,9 @@ class GeneralizationAcrossTime(object):
                 these predictions into a single estimate per sample.
 
         Default: 'cross-validation'
+    predict_shape : int
+        Dimensionality of clf.predict().
+        Default : 1
```

Comment:
can't you get this at fit time? I pretty sure you can avoid this extra parameter.